corretto.hcl

```diff
@@ -32,7 +32,7 @@ channel "stable" {
   version = "11.*"
 }
 
-version "11.0.15.9.1" {
+version "11.0.15.9.1", "11.0.17.8.1" {
```

Comment:
@alecthomas just realizing that there should be no comma here, will this cause problems?

Reply:
I am not able to search or install `corretto` now and wonder if this is why:


```
hermit search corretto

panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x1c8 pc=0x1052f79dc]

goroutine 1 [running]:
github.com/cashapp/hermit/app.Main.func3()
	/home/runner/work/hermit/hermit/app/main.go:149 +0x58
panic({0x1055d9700, 0x105a26500})
	/home/runner/.cache/hermit/pkg/go-1.19.1/src/runtime/panic.go:884 +0x204
github.com/cashapp/hermit/manifest.(*Resolver).Search(0x14000474140, {0x105676ec8, 0x1400017a180}, {0x1400040e700?, 0x1400020a700?})
	/home/runner/work/hermit/hermit/manifest/resolver.go:231 +0x17c
github.com/cashapp/hermit.(*Env).Search(0x140001a3220?, 0x0?, {0x1400040e700, 0x8})
	/home/runner/work/hermit/hermit/env.go:1045 +0x48
github.com/cashapp/hermit/app.(*searchCmd).Run(0x14000306bd0, 0x1400017a180, 0x140001a3220, 0x1?)
	/home/runner/work/hermit/hermit/app/search_cmd.go:87 +0xb8
reflect.Value.call({0x1055ca880?, 0x14000306bd0?, 0x1400020b178?}, {0x105470cc6, 0x4}, {0x14000305500, 0x3, 0x10540e1f0?})
	/home/runner/.cache/hermit/pkg/go-1.19.1/src/reflect/value.go:584 +0x688
reflect.Value.Call({0x1055ca880?, 0x14000306bd0?, 0x1055e2400?}, {0x14000305500?, 0x14000365ea0?, 0x9?})
	/home/runner/.cache/hermit/pkg/go-1.19.1/src/reflect/value.go:368 +0x90
github.com/alecthomas/kong.callMethod({0x105470558, 0x3}, {0x105629c80?, 0x14000306bd0?, 0x3?}, {0x1055ca880?, 0x14000306bd0?, 0x14000469da0?}, 0x10566f2a0?)
	/home/runner/go/pkg/mod/github.com/alecthomas/kong@v0.5.0/callbacks.go:97 +0x3e4
github.com/alecthomas/kong.(*Context).RunNode(0x14000192400, 0x14000336d20, {0x1400020b990, 0x8, 0x8})
	/home/runner/go/pkg/mod/github.com/alecthomas/kong@v0.5.0/context.go:735 +0x460
github.com/alecthomas/kong.(*Context).Run(0x105652ec0?, {0x1400020b990?, 0x5?, 0x5?})
	/home/runner/go/pkg/mod/github.com/alecthomas/kong@v0.5.0/context.go:760 +0x144
github.com/cashapp/hermit/app.Main({{0x1400011c3a0, 0x10}, 0x3, {0x14000124200, 0x3a}, {0x14000109bc0, 0x3, 0x3}, {0x14000318000, 0x13, ...}, ...})
	/home/runner/work/hermit/hermit/app/main.go:279 +0x189c
main.main()
	/home/runner/work/hermit/hermit/cmd/hermit/main.go:28 +0x1cc
```

Searching and installing other packages works.

Reply:
Moving to issue https://github.com/cashapp/hermit-packages/issues/226.